04-KernelModule/Makefile

```diff
@@ -0,0 +1,8 @@
+obj-m+=mymodule.o
+
+BUILD_KERNEL:= /usr/src/linux-headers-$(shell uname -r)
```

Comment:
Better not to hardcode paths. Use environment setup.
Or use BUILD_KEREL ?= xxxxx 
so it will be defined if not defined before.